msr-flows/msbr_compare/serpent_output.py

```diff
@@ -0,0 +1,233 @@
+import serpentTools as st
```

Comment:
I think `pyne` also has a serpent module. But you might have already tried using that.

Reply:
I wasn't aware of that actually, I'm just used to using Serpent Tools. I might have to look into the `pyne` version.